src/ContinuousEnergyNeutron.tex

```diff
@@ -373,10 +386,14 @@ \subsubsection{\textsf{LDLW} \& \textsf{LDLWP} Blocks}\label{sec:LDLWBlock}\labe
 \end{tablenotes}
 \end{ThreePartTable}
 
-\textbf{All locators point to data \emph{relative} to \var{JED} (see \Sectionref{sec:DLWBlock}) in the \XSS\ array.} The \MT\ values are given in the \Block{MTR} for \Block{LDLW} or \Block{MTRP} for \Block{LDLWP}.
+The corresponding \MT\ values are given in the \Block{MTR} for \Block{LDLW} or \Block{MTRP} for \Block{LDLWP}.
+ 
+\textbf{All locators point to data \emph{relative} to \var{JED} (see \Sectionref{sec:DLWBlock}) in the \XSS\ array.}
+
+\subsubsection{\textsf{DLW} \& \textsf{DLWP} \& \textsf{DNED} Blocks}\label{sec:DLWBlock}\label{sec:DLWPBlock}\label{sec:DNEDBlock}
+
+The \Block{DLW} contains secondary energy distributions for all reactions producing secondary neutrons---except for elastic scattering, the \Block{DLWP} contains secondary energy distributions for all photon-producing reactions and the \Block{DNED} contains the energy distributions for the delayed neutrons. The \Block{DLW}, \Block{DLWP} and \Block{DNED} block have the same format. The energy distributions are given starting with a locator, \LOC{C}, which were given in the \Block{LDLW}, \Block{LDLWP} or \Block{LDNED}. The locators are relative to the \var{JED} parameter. The value for \var{JED} and \var{NMT} (the number of reactions or the number of delayed precursor groups) is dependent on whether it is the \Block{DLW}, \Block{DLWP} or \Block{DNED}. These values are given in \Tableref{tab:JED_NMT}.
```

Comment:
The em dash actually makes it confusing to me whether the clause (except for elastic scattering) is supposed to refer to the presence of secondary E distributions for secondary neutrons, or secondary E distributions for photon-producing reactions. I do know what it applies to, but it might be confusing to someone who hasn't spent long nights with the ACE format :)

Reply:
I reworded this slighly.